pkg/types/validation/installconfig_test.go

```diff
@@ -109,6 +109,7 @@ func validBareMetalPlatform() *baremetal.Platform {
 		Hosts: []*baremetal.Host{
 			{
 				Name:           "host1",
+				Role:           "master",
```

Comment:
Is it possible that the config is provided without a defined "Role"/ Do we have defaults when that happens?

Reply:
We don't have defaults (if not in the wizard). In any case the new validation rules for the hosts count will detect now any host without the `role` tag (now filtered per `master` and `worker`).